src/js/ripple/amount.js

```diff
@@ -1,53 +1,55 @@
 // Represent Ripple amounts and currencies.
 // - Numbers in hex are big-endian.
 
-var extend = require('extend');
-var utils = require('./utils');
-var sjcl  = utils.sjcl;
-var bn    = sjcl.bn;
-
-var BigInteger = utils.jsbn.BigInteger;
-
-var UInt160  = require('./uint160').UInt160;
-var Seed     = require('./seed').Seed;
-var Currency = require('./currency').Currency;
-
-//
-// Amount class in the style of Java's BigInteger class
-// http://docs.oracle.com/javase/1.3/docs/api/java/math/BigInteger.html
-//
+var assert    = require('assert');
+var BigNumber = require('bignumber.js');
+var extend    = require('extend');
+var utils     = require('./utils');
+var UInt160   = require('./uint160').UInt160;
+var Seed      = require('./seed').Seed;
+var Currency  = require('./currency').Currency;
+
+utils.configureBigNumber(BigNumber);
```

Comment:
Even if we do that, it won't really solve the problem you described because another file can still require('bignumber.js') without using the wrapper and override the configuration. The only way I can think of to make it safer is to make a new npm package that wraps bignumber.js so that bignumber.js is not a direct dependency of ripple-lib and can't be directly required.